### PR TITLE
add promo timestamps & update other timestamps

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds promo timestamps which is 3 days before the radio shows. After discussion with the team we agreed to increase the sms adverts time period till midnight of that date as we have observed a large number of responses in the evening hours after the advert goes out in the afternoon.

### CODE REVIEW

**Comment by IsaackMwenda** [comment]
> This adds promo timestamps which is 3 days before the radio shows. After discussion with the team we agreed to increase the sms adverts time period till midnight of that date as we have observed a large number of responses in the evening hours after the advert goes out in the afternoon.

@DanielMwendwa feel free to delegate this or any other future prs to @as2388 that you find abit hard to follow through.

**Review by DanielMwendwa — APPROVED** [review]

**Review by as2388 — APPROVED** [review]
